targets/x86/profiles/default/config

```diff
@@ -1698,7 +1432,7 @@ CONFIG_PACKAGE_plugin-gargoyle-i18n-SimplifiedChinese-ZH-CN=m
 CONFIG_PACKAGE_plugin-gargoyle-i18n-Slovak-SK=m
 CONFIG_PACKAGE_plugin-gargoyle-i18n-Spanish-ES=m
 CONFIG_PACKAGE_plugin-gargoyle-initd=m
-CONFIG_PACKAGE_plugin-gargoyle-logread=m
+CONFIG_PACKAGE_plugin-gargoyle-logread=y
```

Comment:
as discussed previously, this should be modularised, not included by default.

Reply:
I think that the logread built-in is necessary so that the log can be fed back in time when something goes wrong.

Reply:
It isn’t included in any other targets, and for consistency I wouldn’t change it. 
The log can be accessed on the command line if needed, and the plugin can be installed later if needed like all other targets.